install_config/persistent_storage/topics/glusterfs_example_basic_external.adoc

```diff
@@ -0,0 +1,43 @@
+. In your inventory file, add `glusterfs` in the `[OSEv3:children]` section to
+enable the `[glusterfs]` group:
++
+----
+[OSEv3:children]
+masters
+nodes
+glusterfs
+----
+
+. (Optional) Include any of the following role variables in the `[OSEv3:vars]`
+section you wish to change:
++
+----
+[OSEv3:vars]
+openshift_storage_glusterfs_namespace=glusterfs <1>
+openshift_storage_glusterfs_name=storage <2>
+----
+<1> The project (namespace) to host the storage pods. Defaults to `glusterfs`.
+<2> A name to identify the GlusterFS cluster, which will be used in resource
+names. Defaults to `storage`.
+
+. Add a `[glusterfs]` section with entries for each storage node that will host
+the GlusterFS storage and include the `glusterfs_ip` and `glusterfs_devices`
+parameters in the form:
++
+----
+<hostname_or_ip> glusterfs_ip=<ip_address> glusterfs_devices='[ "</path/to/device1/>", "</path/to/device2>", ... ]'
+----
++
+For example:
++
+----
+[glusterfs]
+gluster1.example.com glusterfs_ip=192.168.10.11 glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+gluster2.example.com glusterfs_ip=192.168.10.12 glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+gluster3.example.com glusterfs_ip=192.168.10.13 glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+----
++
+Set `glusterfs_devices` to a list of raw block devices that will be completely
```

Comment:
Is this required or optional? Is this closure to let the user know the task is complete?

Reply:
Required. I think I see the confusion. Will reword.